scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -51,6 +51,20 @@ trait Config extends Serializable {
     }
 
   /**
+   * Add files to be localized to the config. Intended to be used by user code.
+   * @param cachedFiles CachedFiles to be added
+   * @return new Config with cached files
+   */
+  def addDistributedCacheFiles(cachedFiles: CachedFile*): Config =
```

Comment:
`CachedFile` itself not depended on `DistributedCacheFile`, it just encapsulates information about qualified URI and has shortcuts for file and path.